### DISCUSSION TITLE
Enterprise Stack creation complete but EC2 Terminated (Externally registered Domain)

### DISCUSSION THREAD

**Discussion by dayekaye** [mozilla/hubs#3301] [Hubs Cloud Questions] [answered]
After the cloud formation successfully ran and the stack is created I see that the EC2 instance is immediately terminated. I do not receive any errors so I am kind of at a loss. I am using Recipe 3 as my Domains are registered outside of Route 52. Looking for any debuging guidance or if this issue has been encountered before.

Thanks

**Comment by jesus-hergueta** [comment]
Hi

Try this #3028

**Comment by dayekaye** [comment]
So I even just tried creating a new instance using a domain name purchased through Route 53 and came up with the same issue. The stack is completed with no errors, but EC2 is terminated.

**Comment by dayekaye** [comment]
small update, the two EC2 instances have an error message that say Server.InternalError

**Comment by dayekaye** [comment, marked as answer]
final update - switched from Virginia to Ohio region and it is working for the Route 53 purchased domains
